diff --git a/gtk/gtksettingsprivate.h b/gtk/gtksettingsprivate.h
index 47f75a6297..fd822c630a 100644
--- a/gtk/gtksettingsprivate.h
+++ b/gtk/gtksettingsprivate.h
@@ -23,8 +23,8 @@
 
 G_BEGIN_DECLS
 
-#define DEFAULT_THEME_NAME      "Adwaita"
-#define DEFAULT_ICON_THEME      "Adwaita"
+#define DEFAULT_THEME_NAME      "io.elementary.stylesheet.blueberry"
+#define DEFAULT_ICON_THEME      "elementary"
 
 void                _gtk_settings_set_property_value_from_rc (GtkSettings            *settings,
                                                               const gchar            *name,
