--- "Projects/Flatpak | Manifests/io.elementary.sdk/build/.flatpak-builder/build/icons/meson.build"	2020-04-26 00:21:25.166387823 -0300
+++ "Projects/Flatpak | Manifests/io.elementary.sdk/build/.flatpak-builder/build/icons/meson.build"	2020-04-26 00:20:57.086493000 -0300
@@ -172,7 +172,7 @@
     endforeach
 endforeach
 
-subdir('cursors')
+#subdir('cursors')
 
 if (get_option ('volume_icons'))
   subdir('volumeicon')
